internal/jobcontainers/jobcontainer.go

```diff
@@ -166,10 +196,29 @@ func (c *JobContainer) CreateProcess(ctx context.Context, config interface{}) (_
 		return nil, errors.New("unsupported process config passed in")
 	}
 
-	// Replace any occurrences of the sandbox mount point env variable in the commandline.
-	// %CONTAINER_SANDBOX_MOUNTPOINT%\mybinary.exe -> C:\C\123456789\mybinary.exe
+	// Replace any occurences of the sandbox mount env variable in the commandline.
+	// For example: %CONTAINER_SANDBOX_MOUNTPOINT%\mybinary.exe -> C:\<rootfslocation>\mybinary.exe.
 	commandLine, _ := c.replaceWithMountPoint(conf.CommandLine)
 
+	// This is to workaround a rather unfortunate outcome with launching a process in a silo that
+	// has bound files.
+	// If a user requested to launch a program at C:\<rootfslocation>\mybinary.exe because they
+	// expect C:\<rootfslocation>\mybinary.exe to exist once the file bindings are done, it actually
+	// won't work. This is because the process is launched on the host first and the
```

Comment:
I think `the process is launched on the host first` is pretty unclear wording. My understanding of the issue is that the exe file is looked up using the parent process's filesystem view, so it can't find it if it's only visible in the container's filesystem view. Suggest we say something like that.

Reply:
Good idea, that's more clear

Reply:
Changed the wording